manifests/web.pp

```diff
@@ -104,7 +104,11 @@
 #   Database port when not using local socket. Ignored for sqlite.
 #
 # [*zabbix_server*]
-#   The fqdn name of the host running the zabbix-server. When single node:
+#   The ip of the host running the zabbix-server. When single node:
```

Comment:
Please adjust the comment. It defaults to an FQDN, not an IP (I guess the code is correct and the comment is wrong, not the other way round?)